pkg/internal/testing/integration/internal/apiserver.go

```diff
@@ -9,7 +9,8 @@ var APIServerDefaultArgs = []string{
 	"--insecure-port={{ if .URL }}{{ .URL.Port }}{{ end }}",
```

Comment:
Since we have `populateAPIServerCerts`, we can cleanup code related to apiserver serving on insecure port.

Reply:
I am not sure we're there yet. And even if we are, I would rather do that in separate PR